controllers/apply-next/docs/validation-rules.md

```diff
@@ -225,12 +241,31 @@ Address field group, consists of:
 
 ### Legal contact role
 
-@TODO
+**Type**: single choice (radio)
+
+| Rule           | Message       |
+| -------------- | ------------- |
+| Required field | Choose a role |
+
+#### Mappings
+
+The legal contact role shows the following choices depending on the organisation type:
```

Comment:
I've added a trello card based to this list so we can refine it. I'm not 100% sure I've interpreted the rules for this correctly as it feels off that there is only one choice of role.